notebooks/open_cytoscape_nb.ipynb

```diff
@@ -0,0 +1,416 @@
+{
```

Comment:
Is this file exploratory code or part of the scripts that you want to include in this pull request?  I see there are some files from the `Visualization` branch that you deleted here, but I'm not sure what the full scope of the file changes here should be.

Reply:
Everything is the notebooks folder is exploratory so we don't need any of those jupyter notebooks in the pull request